ledger-core/virtual/integration/method_test.go

```diff
@@ -911,7 +926,7 @@ func TestVirtual_CallMethodFromConstructor_Unordered(t *testing.T) {
 
 		runnerMock.AddExecutionMock("Foo").AddStart(
```

Comment:
I think that it is better to drop mock of executing B. Just don't resend VCallRequest, check token in the message and generate manually VCallResponse.